testing/environments/docker/kibana/Dockerfile

```diff
@@ -1,45 +1,10 @@
 # Copy of https://github.com/elastic/kibana-docker/blob/master/build/kibana/Dockerfile
-FROM centos:7
-LABEL maintainer "Elastic Docker Team <docker@elastic.co>"
-EXPOSE 5601
+FROM docker.elastic.co/kibana/kibana:7.0.0-alpha1-SNAPSHOT
 
-### Beats specific args ####
-ARG DOWNLOAD_URL
-ARG ELASTIC_VERSION
-ARG CACHE_BUST=1
-ARG IMAGE_FLAVOR=x-pack
+USER root
 
 # Healthcheck create by beats team
 RUN yum install update -y epel-release && yum install -y jq
```

Comment:
@jarpy I remember we discussed this in the past but don't remember the outcome anymore. Now that we have the snapshot docker images this becomes more obvious again. We need jq for the health check. We could probably put something together without jq but regexp but it would make things more complex. Any ideas?

Reply:
Interesting idea, will try that out.